.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
You're right, seems travis uses "Google Compute Engine" for this, and
define it as "Fully-virtualized".

I checked the real "container" env with `sudo: false`, but while it
still offers us to use "Ubuntu 14.04" image, jruby in this case gives
us jruby 1.7.26. So in the end I just fixed the commit message.

Thank you.